lms/djangoapps/courseware/model_data.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, descriptors, course_id, user, select_for_update=False, asides
         self.cache = {}
         self.descriptors = descriptors
         self.select_for_update = select_for_update
+        self.scores = {}
```

Comment:
should this more properly be something like `modules_with_scores`? Just calling it `scores` is a bit confusing